tests/unit/image/test_remote_image_builder.py

```diff
@@ -26,9 +26,10 @@ async def test_image_builder__min_parameters(
     )
 
     expected_storage_build_root = URL(
-        "storage://mycluster/myproject/.builds/mocked-uuid-4"
+        "storage://mycluster/NO_ORG/myproject/.builds/mocked-uuid-4"
```

Comment:
nit: we might need to patch our conftest, to report some org, just to be sure in-org builds are working well